client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/BackfilaParameters.kt

```diff
@@ -15,9 +14,8 @@ import kotlin.reflect.jvm.jvmErasure
 import okio.ByteString
 import okio.ByteString.Companion.encodeUtf8
 
-inline fun <reified T : Any> parametersToBytes(parameters: T): Map<String, ByteString> {
-  val parametersClass = T::class
-
+fun parametersToBytes(parameters: Any): Map<String, ByteString> {
+  val parametersClass = parameters::class
   val map = mutableMapOf<String, ByteString>()
 
   for (property in parametersClass.memberProperties) {
```

Comment:
Without this part of the diff, the added test fails.
Debugging it, it seems due to `parametersClass` always being `Any`, which means this for loop is over an empty `parametersClass.memberProperties`.

Reply:
huh, weird